### DISCUSSION TITLE
doris export 命令是导出当前的快照数据吗

### DISCUSSION THREAD

**Discussion by javaofferss** [apache/doris#38217] [A - General / Q&A]
doris export 命令是导出当前的快照数据吗。比如当我执行export 命令时（状态为EXPORTING），在这个过程中有写入的数据 会在导出的结果中吗？

**Comment by javaofferss** [comment]
来一个大佬解答一下

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子
